packages/babel-core/test/esm-cjs-integration.js

```diff
@@ -52,10 +52,7 @@ describeESM("usage from cjs", () => {
   });
 
   it("eager plugin required", async () => {
-    await expect(run("eager-plugin-required.cjs")).rejects.toThrow(
-      "The `types` export of @babel/core is only accessible from" +
-        " the CommonJS version after that the ESM version is loaded.",
-    );
+    await expect(run("eager-plugin-required.cjs")).resolves.not.toThrow();
```

Comment:
This test should be updated to the inline snapshot similar to other sub tests here.